pymc3/util.py

```diff
@@ -15,10 +15,11 @@
 import re
 import functools
 from typing import List, Dict, Tuple, Union
+import logging
 
 import xarray
 import arviz
-from numpy import asscalar, ndarray
+from numpy import asscalar, ndarray, isfinite, all
```

Comment:
Im being picky but please `import numpy as np`. Its so common to namespace under there that it would make this code easier to read if it follows the pattern

Reply:
Have changed this, will push in a bit